app/src/main/java/edu/uc/coffeens/stocktracker/dto/Stock.kt

```diff
@@ -6,10 +6,10 @@ import com.google.gson.annotations.SerializedName
  * DTO class for a Stock.
  */
 data class Stock(
-    @SerializedName("symbol") var stockTicker: String,
-    @SerializedName("company") var stockCompany: String,
-    @SerializedName("description") var stockDescription: String,
-    @SerializedName("initial_price") var stockPrice: String
+    @SerializedName("symbol") var stockTicker: String?,
+    @SerializedName("company") var stockCompany: String?,
+    @SerializedName("description") var stockDescription: String?,
+    @SerializedName("initial_price") var stockPrice: String?
```

Comment:
I added the question marks to make sure the code can be nulled in the future to be ready against any potential bugs that might occur, as advised by this article https://docs.sonarqube.org/display/SONARQUBE45/Bugs+and+Potential+Bugs .

Reply:
Actually... best to keep them not nullable.  ? opens up the opportunity that they can be null; no ? means they can't .  I'd leave this as it were.